gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -235,29 +268,46 @@ Dump::visit (ClosureExpr &)
 void
 Dump::visit (BlockExpr &block_expr)
 {
-  stream << "BlockExpr: [";
+  stream << "BlockExpr: [\n";
+
   indentation.increment ();
-  stream << std::endl;
   // TODO: inner attributes
+  if (!block_expr.inner_attrs.empty ())
+    {
+      stream << indentation << "inner_attrs: [";
+      for (auto &attr : block_expr.inner_attrs)
```

Comment:
Just add indentation increment and decrement around the for loop (see the example).

Reply:
Fixed.

Reply:
> Just add indentation increment and decrement around the for loop (see the example).

I have updated the [new dump](https://github.com/Rust-GCC/gccrs/pull/1976#issuecomment-1468716299).